src/core/egg-herbie.rkt

```diff
@@ -285,19 +279,29 @@
 
   (*context* (make-debug-context '(x a b c r)))
   (define extended-expr-list
-    (list '(/ (- (exp x) (exp (- x))) 2)
-          '(/ (+ (- b) (sqrt (- (* b b) (* (* 3 a) c)))) (* 3 a))
-          '(/ (+ (- b) (sqrt (- (* b b) (* (* 3 a) c))))
-              (* 3 a)) ;; duplicated to make sure it would still work
+    ; specifications
+    (list '(/ (- (exp x) (exp (neg x))) 2)
+          '(/ (+ (neg b) (sqrt (- (* b b) (* (* 3 a) c)))) (* 3 a))
+          '(/ (+ (neg b) (sqrt (- (* b b) (* (* 3 a) c)))) (* 3 a))
```

Comment:
The comment on why we have a duplicate is bizarre but maybe keep it?

Reply:
Not sure. I just copied what we had